app/renderer/components/navigation/navigationBar.js

```diff
@@ -266,13 +270,35 @@ class NavigationBar extends React.Component {
   }
 }
 
+// Create 25x25 squares and place the buttons at the center of each container
+const buttonContainer = {
+  border: `1px solid ${globalStyles.color.urlBarOutline}`,
+  borderRadius: globalStyles.radius.borderRadiusURL,
+  boxSizing: 'border-box',
+  display: 'flex',
+  alignItems: 'center',
+  height: globalStyles.navigationBar.urlbarForm.height,
+  width: globalStyles.navigationBar.urlbarForm.height,
+  minHeight: globalStyles.navigationBar.urlbarForm.height,
+  minWidth: globalStyles.navigationBar.urlbarForm.height,
+  WebkitAppRegion: 'no-drag'
+}
```

Comment:
those lines are moved to commonStyles.js on https://github.com/brave/browser-laptop/pull/9299/commits/573f562c3d682d054db6bd5fb8dbbe81f4b55618